satellite/overlay/selection_test.go

```diff
@@ -83,11 +98,17 @@ func TestMinimumDiskSpace(t *testing.T) {
 		require.NoError(t, err)
 
 		// request 2 nodes, expect success
-		_, err = planet.Satellites[0].Overlay.Service.FindStorageNodes(ctx, overlay.FindStorageNodesRequest{
-			MinimumRequiredNodes: 2,
-			RequestedCount:       2,
-		})
+		n1, err = planet.Satellites[0].Overlay.Service.FindStorageNodes(ctx, req)
+		require.NoError(t, err)
+		require.Equal(t, 2, len(n1))
+		n2, err = saOverlay.Service.FindStorageNodesWithPreferences(ctx, req, &nodeConfig)
+		require.NoError(t, err)
+		require.Equal(t, len(n1), len(n2))
```

Comment:
nit/nbd - you can use `require.Len` instead of `require.Equal` here which might make some of these lines slightly easier to read

Reply:
i like the equal better, that seems more readable to me lol